From: Jiri Kosina <jkosina@suse.cz>
Subject: KDB: fix build in XFS code
Patch-mainline: not applicable

This function is not used anywhere, so we can just drop it.

Signed-off-by: Jiri Kosina <jkosina@suse.cz>

Index: linux-3.0-SLE11-SP2/fs/xfs/linux-2.6/xfs_buf.c
===================================================================
--- linux-3.0-SLE11-SP2.orig/fs/xfs/linux-2.6/xfs_buf.c
+++ linux-3.0-SLE11-SP2/fs/xfs/linux-2.6/xfs_buf.c
@@ -1889,7 +1889,7 @@ xfs_buf_terminate(void)
 	destroy_workqueue(xfslogd_workqueue);
 	kmem_zone_destroy(xfs_buf_zone);
 }
-
+/* Breaks build, probably not needed at all
 #ifdef CONFIG_KDB_MODULES
 struct list_head *
 xfs_get_buftarg_list(void)
@@ -1897,3 +1897,4 @@ xfs_get_buftarg_list(void)
 	return &xfs_buftarg_list;
 }
 #endif
+*/
Index: linux-3.0-SLE11-SP2/fs/xfs/linux-2.6/xfs_buf.h
===================================================================
--- linux-3.0-SLE11-SP2.orig/fs/xfs/linux-2.6/xfs_buf.h
+++ linux-3.0-SLE11-SP2/fs/xfs/linux-2.6/xfs_buf.h
@@ -339,9 +339,11 @@ extern void xfs_wait_buftarg(xfs_buftarg
 extern int xfs_setsize_buftarg(xfs_buftarg_t *, unsigned int, unsigned int);
 extern int xfs_flush_buftarg(xfs_buftarg_t *, int);
 
+/*
 #ifdef CONFIG_KDB_MODULES
 extern struct list_head *xfs_get_buftarg_list(void);
 #endif
+*/
 
 #define xfs_getsize_buftarg(buftarg)	block_size((buftarg)->bt_bdev)
 #define xfs_readonly_buftarg(buftarg)	bdev_read_only((buftarg)->bt_bdev)
